### PR TITLE
update to miniforge3

### LINKED ISSUE #14: Mambaforge (Discouraged as of September 2023)
We should at least update the develop container

similiar to https://github.com/Open-MSS/mss-install/issues/18

### PR DESCRIPTION
**Pull request by ReimarBauer**
closes #14

### CODE REVIEW

**Comment by ReimarBauer** [comment]
@joernu76 that works locally, can you have a check please

File: Dockerfile [hunk 1]
```diff
@@ -31,22 +31,22 @@ RUN  export DEBIAN_FRONTEND=noninteractive \
 ENV PYTHONPATH="/srv/mss:/root/mss"
 
 # Install requirements, fetched from the specified branch
-RUN wget -O /meta.yaml -q https://raw.githubusercontent.com/Open-MSS/MSS/${BRANCH}/localbuild/meta.yaml \
+RUN  wget -O /meta.yaml -q https://raw.githubusercontent.com/Open-MSS/MSS/${BRANCH}/localbuild/meta.yaml \
```
Comment (matrss) [inline Dockerfile]: Does this change have a reason or was it by mistake?

File: Dockerfile [hunk 2]
```diff
@@ -39,14 +39,14 @@ RUN wget -O /meta.yaml -q https://raw.githubusercontent.com/Open-MSS/MSS/${BRANC
...
-  && conda clean --all \
+  && mamba clean -y --all \
   && rm reqs.txt \
-  && cp /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh 
+  && cp /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
+  && cp /opt/conda/etc/profile.d/mamba.sh /etc/profile.d/mamba.sh
 
 # execute /etc/profile also in non-interactive use
-ENV BASH_ENV /etc/profile.d/conda.sh
+ENV BASH_ENV /etc/profile.d/mamba.sh
```
Comment (joernu76) [inline Dockerfile:50]: when I install miniforge, I have to execute conda.sh and mamba.sh for everything to work...

**Review by joernu76 — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (ReimarBauer) [reply Dockerfile:50]: @joernu76 I tried locally with BRANCH set to develop and

```
docker run -it $(docker build -q .)
(base) root@e03e9a3fcf42:/# mamba activate mss-develop-env
```

Which failure have you seen?

Comment (joernu76) [reply Dockerfile:50]: I *had* issues with an older version of the software. This was just a caveat. I approved the MR:

**Pull request merged**